### PR TITLE
Feat: quote, Todo

### PR DESCRIPTION
**Pull request by yashpatel1797**
Feature
-Todo
- quote on every reload

### CODE REVIEW

**Review by bishnoimukesh — APPROVED** [review]
Everything looks good
Overall clean UI 
Good file structure

File: src/Reducer/todoReducer.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { v4 as uuid } from "uuid"
+
+const todoReducer = (state, { type, payload }) => {
+    switch (type) {
+        case "ADD_TODO":
+            return { ...state, todoList: [...state.todoList, { id: uuid(), task: payload, isComplete: false }] }
+        case "TODO_STATUS":
+            return { ...state, todoList: state.todoList.map(item => payload.id === item.id ? { ...item, isComplete: !item.isComplete } : item) }
+        case "TODO_DELETE":
+            console.log("res");
```
Comment (ivinayakg) [inline src/Reducer/todoReducer.js]: remove this

File: src/components/Quote.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React, { useEffect, useState } from 'react'
+import axios from "axios";
+const Quote = () => {
```
Comment (ivinayakg) [inline src/components/Quote.js:3]: nice feature

File: src/components/Todo.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React, { useState } from 'react'
+import { TodoModal } from './TodoModal'
+const Todo = () => {
+    const [showModal, setShowModal] = useState("")
```
Comment (ivinayakg) [inline src/components/Todo.js:4]: very clean and no bs here, great

**Review by ivinayakg — APPROVED** [review]
Great Work 🙌

**Pull request merged**